src/Compilers/Core/Portable/Text/LargeText.cs

```diff
@@ -238,7 +238,8 @@ private SegmentedList<int> ParseLineStarts()
             var position = 0;
             var index = 0;
             var lastCr = -1;
-            var list = new SegmentedList<int>();
+            // Initial line capacity estimated at 64 chars / line
+            var list = new SegmentedList<int>(Length / 64);
```

Comment:
you might want to mention this was based on looking at roslyn files and coming up with a reasonable balance.

Reply:
Done!

Reply:
You say that.  And yet...

Reply:
Guess I was in too big of a hurry to get home to push the button

Reply:
Just want to add that I really appreciate context like this for magic numbers.